bundles/cis-k8s-v1.5.1/5.1.1_restrict-clusteradmin-rolebindings.yaml

```diff
@@ -55,11 +55,9 @@ spec:
     # for Config Sync reconciler service account
     - kind: ServiceAccount
       name: reconciler-manager
-      namespace: config-management-system
```

Comment:
Is this OK? This would allow SAs with the name `reconciler-manager` to become clusteradmin is namespaces other than config-management-system.

Reply:
Thanks - This is to align with the current template fields: https://cloud.google.com/anthos-config-management/docs/latest/reference/constraint-template-library#k8srestrictrolebindings